MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -318,10 +319,13 @@ - (void)detachAVPlayer {
 }
 
 - (void)monitorAVPlayerItem {
-    if (!_automaticVideoChange) {
+    if (!_automaticVideoChange && !_didTriggerManualVideoChange) {
         return;
     }
     if (_playerItem) {
+        if (_didTriggerManualVideoChange) {
```

Comment:
Is this actually more performant in some way, or is this for readability? either way it's fine

Reply:
I did this for readability, I don't know of any performance difference

Reply:
👍